logstash-core/src/main/java/org/logstash/Event.java

```diff
@@ -264,16 +264,16 @@ public Event clone() throws CloneNotSupportedException {
     }
 
     public String toString() {
-        // TODO: (colin) clean this IOException handling, not sure why we bubble IOException here
-        try {
-            return (getTimestamp().toIso8601() + " " + this.sprintf("%{host} %{message}"));
-        } catch (IOException e) {
-            String host = (String)this.data.get("host");
-            host = (host != null ? host : "%{host}");
+        Object f = this.getField("host");
```

Comment:
It think given the very narrow scope & simplicity of the 4 lines involved in sharing this temporary variable it does not really matter unless I am missing a condition where this could cause a problem in this context? Is there a condition where this code could not be correct?

But since this seems to be causing difficulties in assessing this bit of code I'll go ahead and  change it to avoid any other potential confusion.